ibis/backends/oracle/__init__.py

```diff
@@ -242,7 +243,7 @@ def _metadata(self, query: str) -> Iterable[tuple[str, dt.DataType]]:
             t.c.data_type,
             t.c.data_precision,
             t.c.data_scale,
-            (t.c.nullable == "Y").label("nullable"),
+            case([(t.c.nullable == "Y", True)], else_=False).label("nullable"),
```

Comment:
```suggestion
            case((t.c.nullable == "Y", True), else_=False).label("nullable"),
```

`case` is variadic now.